accurest-core/src/test/groovy/io/codearte/accurest/dsl/WiremockGroovyDslSpec.groovy

```diff
@@ -313,6 +315,168 @@ class WiremockGroovyDslSpec extends WiremockSpec {
+		given:
+			GroovyDsl groovyDsl = GroovyDsl.make {
+				request {
+					method 'GET'
+					urlPath($(client("users"), server("items"))) {
+						queryParameters {
+							parameter 'limit': $(client(equalTo("20")), server(containing("10")))
+							parameter 'offset': containing("10")
+							parameter 'filter': "email"
+							parameter 'sort': ~/^[0-9]{10}$/
+							parameter 'search': $(client(notMatching(~/^\/[0-9]{2}$/)), server(containing("10")))
+							parameter 'age': notMatching("^\\w*\$")
+							parameter 'name': matching("Denis.*")
+						}
+					}
+				}
+				response {
+					status 200
+				}
+			}
+		when:
+			def json = toWiremockClientJsonStub(groovyDsl)
+		then:
+			parseJson(json) == parseJson('''
+			{
+				"request": {
+					"method": "GET",
+					"urlPath": "users",
+					"queryParameters": {
+					  "offset": {
+						"contains": "10"
+					  },
+					  "limit": {
+						"equalTo": "20"
+					  },
+					  "filter": {
+						"equalTo": "email"
+					  },
+					  "sort": {
+                        "matches": "^[0-9]{10}$"
+                      },
+                      "search": {
+                        "doesNotMatch": "^/[0-9]{2}$"
+                      },
+                      "age": {
+                        "doesNotMatch": "^\\\\w*$"
+                      },
+                      "name": {
+                        "matches": "Denis.*"
+                      }
+					}
+				},
+				"response": {
+					"status": 200,
+				}
+			}
+			''')
+		and:
+			stubMappingIsValidWiremockStub(json)
+	}
+
+	def "should generate request with urlPath for client side"() {
+		given:
+		GroovyDsl groovyDsl = GroovyDsl.make {
+			request {
+				method 'GET'
+				urlPath $(client("boxes"), server("items"))
+			}
+			response {
+				status 200
+			}
+		}
+		when:
+		def json = toWiremockClientJsonStub(groovyDsl)
+		then:
+		parseJson(json) == parseJson('''
+			{
+				"request": {
+					"method": "GET",
+					"urlPath": "boxes"
+				},
+				"response": {
+					"status": 200,
+				}
+			}
+			''')
+		and:
+		stubMappingIsValidWiremockStub(json)
+	}
+
+	def "should generate simple request with urlPath for client side"() {
+		given:
+		GroovyDsl groovyDsl = GroovyDsl.make {
+			request {
+				method 'GET'
+				urlPath "boxes"
+			}
+			response {
+				status 200
+			}
+		}
+		when:
+		def json = toWiremockClientJsonStub(groovyDsl)
+		then:
+		parseJson(json) == parseJson('''
+			{
+				"request": {
+					"method": "GET",
+					"urlPath": "boxes"
+				},
+				"response": {
+					"status": 200,
+				}
+			}
+			''')
+		and:
+		stubMappingIsValidWiremockStub(json)
+	}
+
+	def "should generate request with url and queryParameters for client side"() {
+		given:
+		GroovyDsl groovyDsl = GroovyDsl.make {
+			request {
+				method 'GET'
+				url(regex(/users\/[0-9]*/)) {
+					queryParameters {
+						parameter 'age': notMatching("^\\w*\$")
+						parameter 'name': matching("Denis.*")
```

Comment:
If we have regular expressions on the client side then we have to enforce a concrete value on the server side. Otherwise we won't be able to build the URL. WDYT?